src/components/NavBar.js

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import logo from '../assets/banana-01.png';
 import { useHistory, Link } from 'react-router-dom';
+import { AuthContext } from '../context/AuthContext';
 
 function NavBar() {
+  const { isAuth, logout } = useContext(AuthContext);
   const history = useHistory();
```

Comment:
waarom heb je useHistory gebruikt want er is nieuw hook useNavigate. Allebei werkt zelfde maar nieuw hook is beter om te gebruiken, denk ik.

De rest vind ik helemaal prima. Heel goed gedaan.

Reply:
Ik had dat pas in de gaten toen ik de les van Elwyn had gevolgd. Ik heb nu inderdaad useNavigate gebruikt in men eindopdracht. Bedankt! :)